build.gradle.kts

```diff
@@ -121,8 +121,11 @@ dependencies {
     implementation("org.openrewrite:rewrite-yaml")
     implementation("org.openrewrite:rewrite-gradle")
     implementation("org.openrewrite:rewrite-maven")
+    implementation("org.openrewrite:rewrite-templating:$rewriteVersion")
     implementation("org.openrewrite.recipe:rewrite-java-dependencies:${rewriteVersion}")
     implementation("org.openrewrite.gradle.tooling:model:${rewriteVersion}")
+    implementation("org.springframework.data:spring-data-mongodb:2.2.13.RELEASE")
+    implementation("org.mongodb:mongo-java-driver:3.12.14")
```

Comment:
I think these can be `compileOnly`, as the recipes we generate will only contain references in Strings; might need to be checked against a project in practice.

Reply:
Did you check this against a small real project? Just to be sure that part was not missed.

Reply:
Will do!